changelogs/fragments/4916-opentelemetry-ini-options.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - opentelemetry callback plugin - allow configuring opentelementry callback via config file (https://github.com/ansible-collections/community.general/pull/4916)
```

Sorry, one period is still missing :-)
```suggestion
  - opentelemetry callback plugin - allow configuring opentelementry callback via config file (https://github.com/ansible-collections/community.general/pull/4916).
```

---

pffff ;-) (done)